### ISSUE TITLE
EFCore 2.0.1 does not build in UWP 

### ISSUE THREAD

**Issue by ridomin** [aspnet/EntityFrameworkCore#11223]
Empty UWP project with reference to EFCore 2.0.1 fails with 

```
 XamlCompiler error WMC1006: Cannot resolve Assembly or Windows Metadata file 'Type universe cannot resolve assembly: Remotion.Linq, Version=2.1.0.0, Culture=neutral, PublicKeyToken=fee00910d6e5f53b.'
```

## Repro Steps.
- OS: Windows Insider RS4
- RS4 SDK installed
- VS 2017 15.6.1
- Create UWP project (TPV/TPMinV=16299, UWP MetaPacakge=6.0.8)
- Install-Package Microsoft.EntityFrameworkCore (latest version is 2.0.1)
- Build

**Observed**: Build fails with the Remotion.Linq error shown above
**Expected**: build succeeds as described in https://docs.microsoft.com/en-us/ef/core/get-started/uwp/getting-started

**Comment by divega** [comment]
[Microsoft.EntityFrameworkCore 2.0.1](https://www.nuget.org/packages/Microsoft.EntityFrameworkCore/2.0.1) depends on [Remotion.Linq 2.1.1](https://www.nuget.org/packages/Remotion.Linq/2.1.1), which targets .NET Standard 1.0. 

Remotion.Linq.dll assembly version in that package is in fact 2.1.0.0.

 Could this be an issue with RS4 SDK?

**Comment by nvborisenko** [comment]
It seems this is fixed in 2.1.0-preview

**Comment by ajcvickers** [comment]
@divega Can this be moved out of the 2.1 milestone. (We are trying to make 2.1 only include issues that we must fix for 2.1)

**Comment by ridomin** [comment]
This was a bug in VS 15.6 that has been serviced in 15.6.5
[588165](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/588165) not related to EFCore.
